fixieai/agents/api.py

```diff
@@ -20,6 +22,28 @@ class Embed:
     # upload an Embed object to Fixie and generate a URL.
     uri: str
 
+    @property
+    def content(self) -> bytes:
```

Comment:
Since the other one is 'text', can we use 'bytes' or 'raw_bytes' or something instead? 'content' is kind of generic.

Reply:
Requests uses .text and .content and it seems to work pretty well (especially since we have .content_type to describe the bytes).